tests/utils/UnionFindTest.cpp

```diff
@@ -14,7 +13,7 @@ namespace cura
  */
 class UnionFindTest : public testing::Test
 {
-public:
+  public:
```

Comment:
Indent with 4 spaces.

Reply:
Will change the `.clang-format` to reflect that